static/src/js/components/TemporalDisplay/__tests__/TemporalSelectionDropdown.test.js

```diff
@@ -291,4 +349,219 @@ describe('TemporalSelectionDropdown component', () => {
 
     expect(onChangeQueryMock).toHaveBeenCalledTimes(1)
   })
+
+  test('onMetricsTemporalFilter is called on Apply', async () => {
+    const onMetricsTemporalFilterMock = jest.fn()
+    const onChangeQueryMock = jest.fn()
+    const user = userEvent.setup()
+
+    setup({
```

Comment:
```suggestion
    const {user } = setup({
```
We can do the `const user = userEvent.setup()` in the setup and just pull it out as a prop for all these

Reply:
This doesn't work because the `setup()` method in this test doesn't return `user`. In this file it returns:
`return render(
    <Provider store={store}>
      <TemporalSelectionDropdown {...props} />
    </Provider>
  )`
  
  This works in other test files though like `DatepickerContainer.test.js` where `setup()` returns:
  `return {
    props,
    user
  }`
  
  I think we could probably rewrite the setup method in this test, but that would involve rewriting all the other tests as well.